docs/reference/query-dsl/query-string-syntax.asciidoc

```diff
@@ -55,6 +55,24 @@ match the query string `"a* b* c*"`.
 
 [WARNING]
 =======
+Pure wildcards `\*` are rewritten to <<query-dsl-exists-query,`exists`>> queries for efficiency.
+As a consequence documents with an empty value on a `text` or a `keyword` field
+like below would match the wildcard `"field:*"`:
```

Comment:
This might be easier to understand if the order of the sentence was changed a bit: "As a consequence, the wildcard field:* would match documents with an emtpy [...] like the following: ..ex1.. but the query would **not** match the following: ..ex2..